### PR TITLE
Payment History API (2/2)

### PR DESCRIPTION
**Pull request by amiecorso**
## High Level Overview of Change

Implement a public API for `paymentHistoy`. 
(1/2) is https://github.com/xpring-eng/Xpring-JS/pull/293
Analogue in Swift SDK: https://github.com/xpring-eng/XpringKit/pull/148

### Context of Change

Expose new `paymentHistory` method as a public API in XRPClient.  Wire an integration test for this functionality. 

Update CHANGELOG/README for new functionality. 

### Type of Change

- [x] New feature (non-breaking change which adds functionality)
- [x] Documentation Updates

## Before / After

Payment history works. 

## Test Plan

New integration test in CI.

### CODE REVIEW

**Comment by amiecorso** [comment]
My integration test is failing with an error coming from gRPC: 
```
Error: 3 INVALID_ARGUMENT: Could not decode account
    at Object.exports.createStatusError (node_modules/grpc/src/common.js:91:15)
    at Object.onReceiveStatus (node_modules/grpc/src/client_interceptors.js:1209:28)
    at InterceptingListener._callNext (node_modules/grpc/src/client_interceptors.js:568:42)
    at InterceptingListener.onReceiveStatus (node_modules/grpc/src/client_interceptors.js:618:8)
    at callback (node_modules/grpc/src/client_interceptors.js:847:24)
```
Any ideas?

File: README.md [hunk 1]
```diff
@@ -184,9 +184,22 @@ const transactionStatus = xrpClient.getPaymentStatus(transactionHash); // Transa
 **Note:** The example transactionHash may lead to a "Transaction not found." error because the TestNet is regularly reset, or the accessed node may only maintain one
 month of history.  Recent transaction hashes can be found in the XRP Ledger Explorer: https://livenet.xrpl.org/
 
+#### Payment history
+
+An `XRPClient` can return payments to and from an account.
```
Comment (keefertaylor) [inline README.md]: Maybe "can return a list of payments to and from an account"

I know you're probably copying me ...

File: src/xrp-client.ts [hunk 2]
```diff
@@ -101,6 +102,19 @@ class XRPClient implements XRPClientInterface {
   public async accountExists(address: string): Promise<boolean> {
     return this.decoratedClient.accountExists(address)
   }
+
+  /**
+   * Return the history of payments for the given account.
+   * Note: This method only works for payment type transactions, see: https://xrpl.org/payment.html
```
Comment (keefertaylor) [inline src/xrp-client.ts]: +1 line after method summary.

File: test/xrp-client-integration-test.ts [hunk 3]
```diff
@@ -130,4 +130,14 @@ describe('Xpring JS XRPClient Integration Tests', function(): void {
     const doesExist = await xrpClient.accountExists(recipientAddress)
     assert.equal(doesExist, true)
   })
+
+  it('Payment History - rippled', async function(): Promise<void> {
+    this.timeout(timeoutMs)
+
+    const payments = await xrpClient.paymentHistory(recipientAddress)
+    // TODO(amiecorso): is this a sufficient integration test?
```
Comment (keefertaylor) [inline test/xrp-client-integration-test.ts]: We could check the length is greater than 1. Presumably the send test just ran above this one (which generated a payment transaction) so we should always get at least one result back.

File: README.md [hunk 4]
```diff
@@ -184,9 +184,22 @@ const transactionStatus = xrpClient.getPaymentStatus(transactionHash); // Transa
...
 month of history.  Recent transaction hashes can be found in the XRP Ledger Explorer: https://livenet.xrpl.org/
 
+#### Payment history
+
+An `XRPClient` can return a list of payments to and from an account.
+
+```javascript
+const { XRPClient } = require("xpring-js");
+
+const remoteURL = "alpha.test.xrp.xpring.io:50051"; // TestNet URL, use alpha.xrp.xpring.io:50051 for MainNet
```
Comment (intelliot) [inline README.md]: nit: I think we capitalize these as `Testnet` and `Mainnet`

Comment (amiecorso) [reply README.md]: standardized throughout both READMEs!

**Pull request merged**